package.json

```diff
@@ -1,105 +1,71 @@
 {
-  "name": "greenfield-template",
-  "version": "0.0.1",
-  "description": "BCGov greenfield template. For reference, testing and new projects.",
-  "main": "index.js",
+  "name": "nr-sample-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.10.0",
+    "@emotion/styled": "^11.10.0",
+    "@mui/material": "^5.10.2",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@typescript-eslint/parser": "5.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "jest": "^29.0.0",
+    "node-sass": "^7.0.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "5.0.1",
+    "react-test-renderer": "^18.2.0",
+    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/v1.2.6/shared-components-1.2.6.tgz",
+    "styled-components": "^5.3.5",
+    "ts-jest": "^28.0.8",
+    "typescript": "*",
+    "web-vitals": "^2.1.0"
+  },
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "nest build",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "lint:staged": "./node_modules/.bin/lint-staged",
-    "make-badges": "istanbul-badges-readme --logo=jest --exitCode=1",
-    "make-badges:ci": "npm run make-badges -- --ci",
+    "start": "react-scripts start",
+    "build": "tsc -p tsconfig.build.json",
     "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/bcgov/greenfield-template.git"
```

Comment:
I think we can keep repository info, including bellow sections as keywords, author, license, bugs and homepage. For "author" we can have "Encora Team" I think.

Reply:
Agree, adding the info there.